### DISCUSSION TITLE
Easy Colab Link?

### DISCUSSION THREAD

**Discussion by arlojeremy** [runew0lf/RuinedFooocus#156] [Ideas]
I have a crap gpu and I have Colab pro anyways. Most AI image gen stuff seems to work okay, including the original Fooocus, but there's no easy link on the main page to do this with Ruined. I'd prefer to do it in colab since generating a 1024 x 1024 SDXL image on my 980ti 6gb take 2-5 minutes even on Forge. Without forge it's like 7 minutes minimum.

I'll admit I'm a super noob so maybe it's something I could do myself, but...

Anyways, thanks!

**Comment by kengun2077** [comment]
!pip install xformers==0.0.27
#!sudo apt-get install python3-gi python3-gi-cairo python3-gi-dbg python-gi-dev python3-blockdev python3-gst-1.0
!apt -y install -qq aria2

%cd /content
!git clone https://github.com/runew0lf/RuinedFooocus.git

!pip install -r /content/RuinedFooocus/requirements_versions.txt

!aria2c --console-log-level=error -c -x 16 -s 16 -k 7M https://huggingface.co/bluepen5805/blue_pencil-XL/resolve/main/blue_pencil-XL-v7.0.0.safetensors -d /content/RuinedFooocus/models/checkpoints -o blue_pencil-XL-v700.safetensors



!python /content/RuinedFooocus/entry_with_update.py  --share 



----------------------

Please use `pip install pygobject` if you want playsound to run more efficiently.
!apt-cache search python gobject
!sudo apt-get install python3-gi python3-gi-cairo python3-gi-dbg python-gi-dev python3-blockdev python3-gst-1.0

If you don't want change the pytorch version in Google Colab
Please use  !pip list -v  find the torch's version 

i.e. 
xformers version 0.0.27 that runs on torch 2.3.1+cu121
